app/assets/javascripts/views/ingredients.coffee

```diff
@@ -19,6 +20,7 @@ class Mixtio.Views.Ingredients extends Backbone.View
         consumableTypes: @consumableTypes
         kitchens: @kitchens
         units: @units
+        forRecipe: @forRecipe
```

Comment:
I think this option will also need to be added to the `add` method in this file. This should cause a couple of the tests to fail in [spec/features/consumable_types_spec.rb](https://github.com/sanger/mixtio/pull/107/files#diff-7ea8fc78de024d91e74cbd51a7a3f400R57) as they look for the "number" input.